Actions/DetermineProjectsToBuild/action.yaml

```diff
@@ -35,8 +38,7 @@ runs:
       run: |
         $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
         try {
-          $changedFiles = ${{ github.action_path }}/GetModifiedFiles.ps1 -token ${{ github.token }}
```

Comment:
If we could follow the original naming strategy and structure of the action yaml file, then you could add a test like this one: https://github.com/microsoft/AL-Go/pull/839/files#diff-d899fcc107a4ee4a13082c5f9daa7c865a2295160b6dba20ea8d45af5cdf1df9 - and ensure that parameters and everything matches the powershell script behind the action.
Unless there is a very good reason not to do that, I think we should

Reply:
My reasoning (back then when the action was created) was that half of the action parameters aren't actually required and the actions shouldn't be called with different values than the default (referring to `github.workspace` and `github.token`).

The script requires them because, I suppose, there is no good way to fetch these values directly in the script.

Other than that, I also believe we should take another look at the "YAML matches PS" approach. The more I read about GitHub composite actions, the more I think there're YAML goodies we don't take advantage of because of this approach.

Another example where I went rogue some time ago is the `Download Project Dependencies` action: it has two steps, not one. 😨